app/controllers/api/v1/rooms_controller.rb

```diff
@@ -106,6 +106,23 @@ def find_room
       def room_params
         params.require(:room).permit(:name, :user_id, :presentation)
       end
+
+      # Generates an access code on rooms#create if the access code room configuration is enabled
+      def generate_access_code
```

Comment:
I don't think that is indeed generating access codes if they are forced enabled on rooms creation,
I think it's auto generating access codes for created rooms all time,

Reply:
That's odd, I tested either.
Maybe, I missed something.